data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -105,31 +105,22 @@ then
 		--volume "$PWD:/assets:z" \
 		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
 		/usr/bin/cluster-etcd-operator render \
-		--etcd-ca=/assets/tls/etcd-ca-bundle.crt \
-		--etcd-ca-key=/assets/tls/etcd-signer.key \
-		--manifest-etcd-image="${MACHINE_CONFIG_ETCD_IMAGE}" \
-		--asset-input-dir=/assets/tls \
+		--asset-input-dir=/dev/null/deprecated \
+		--config-output-file=/dev/null/deprecated \
 		--asset-output-dir=/assets/etcd-bootstrap \
-		--config-output-file=/assets/etcd-bootstrap/config \
+		--cluster-configmap-file=/assets/manifests/cluster-config.yaml \
+		--etcd-image="${MACHINE_CONFIG_ETCD_IMAGE}" \
```

Comment:
For posterity, I am noting here that the `--manifest-etcd-image` flag has been replaced by the `--etcd-image` flag.